posts/2021-11-02-authenticating-requests-using-saml.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Authenticating Vendia Share Requests Using SAML'
+description: Taking advantage of existing SAML Identity Providers when using Vendia Share
+longDescription: >
+    Vendia customers may have already made investments in identity services to allow 
+date: '2021-11-02'
+authors:
+  - Brian McNamara
+categories:
+  - architecture
+---
+
+Vendia Share is a Universal Application platform that accelerates the development of multi-party applications. Each party, or participant, has access to their own node within a Universal Application (Uni), allowing them to control a number of settings. One of the settings available is the authentication mechanisms API clients can use to access their node.
+
+Vendia Share allows participants to authenticate node API requests using several methods:
```

Comment:
Any reason not to list all options here?

Reply:
Nope - just a miss on my part. Addressed in latest push.